include/spirv-tools/libspirv.hpp

```diff
@@ -117,6 +119,13 @@ class SpirvTools {
   bool Validate(const uint32_t* binary, size_t binary_size,
                 const ValidatorOptions& options) const;
 
+  bool Link(const std::vector<std::vector<uint32_t>>& binaries,
+            std::vector<uint32_t>& linked_binary,
+            uint32_t options = kDefaultLinkerOptions) const;
```

Comment:
Why should I drop the options argument from the C++ version of the functions? Is there another way to pass the linking options? (I mostly followed the API used by the assembler and disassembler.)